packages/theme-data/package.json

```diff
@@ -22,13 +22,15 @@
     "build:core": "hig-scripts-build",
     "build:themes": "babel-node scripts/buildThemes",
     "build:theme-artifacts": "node scripts/buildThemeArtifacts.js",
-    "lint": "hig-scripts-lint"
+    "lint": "hig-scripts-lint",
+    "release": "hig-scripts-release"
   },
   "devDependencies": {
     "@hig/babel-preset": "^0.1.1",
     "@hig/eslint-config": "^0.1.0",
     "@hig/jest-preset": "^0.1.0",
     "@hig/scripts": "^0.1.2",
+    "@hig/semantic-release-config": "^0.1.0",
```

Comment:
The configuration dependency is provided via `@hig/scripts` and configured via a property. See the other packages for reference https://github.com/Autodesk/hig/blob/development/packages/button/package.json#L48-L50